--- model_orig.py	2021-11-18 11:50:16.285057063 +0100
+++ model.py	2021-11-18 11:48:38.613273338 +0100
@@ -163,9 +163,12 @@
     # This block is to ensure benchmark timings are accurate. Some blocking is
     # already happening when computing get_confidence_metrics, and this ensures
     # all outputs are blocked on.
-    jax.tree_map(lambda x: x.block_until_ready(), result)
-    result.update(
-        get_confidence_metrics(result, multimer_mode=self.multimer_mode))
-    logging.info('Output shape was %s',
-                 tree.map_structure(lambda x: x.shape, result))
+    try:
+        jax.tree_map(lambda x: x.block_until_ready(), result)
+        result.update(
+            get_confidence_metrics(result, multimer_mode=self.multimer_mode))
+        logging.info('Output shape was %s',
+                     tree.map_structure(lambda x: x.shape, result))
+    except KeyError:
+        logging.info('Representations only?')
     return result, recycles
