--- modules_orig.py	2021-11-18 09:59:25.411784193 +0100
+++ modules.py	2021-11-20 14:32:45.240410673 +0100
@@ -139,10 +139,11 @@
   def __call__(self,
                ensembled_batch,
                non_ensembled_batch,
-               is_training,
+               is_training=False,
                compute_loss=False,
                ensemble_representations=False,
-               return_representations=False):
+               return_representations=True):
 
     num_ensemble = jnp.asarray(ensembled_batch['seq_length'].shape[0])
 

@@ -284,7 +287,8 @@
       is_training,
       compute_loss=False,
       ensemble_representations=False,
-      return_representations=False):
+      return_representations=True):
     """Run the AlphaFold model.
 
     Arguments: