--- modules.py	2022-01-13 19:28:35.861189933 +0100
+++ modules.backup.py	2022-01-13 19:24:15.217330187 +0100
@@ -142,7 +142,7 @@
                is_training,
                compute_loss=False,
                ensemble_representations=False,
-               return_representations=True):
+               return_representations=False):
 
     num_ensemble = jnp.asarray(ensembled_batch['seq_length'].shape[0])
 
@@ -284,7 +284,7 @@
       is_training,
       compute_loss=False,
       ensemble_representations=False,
-      return_representations=True):
+      return_representations=False):
     """Run the AlphaFold model.
 
     Arguments:
